yt_idefix/io.py

```diff
@@ -118,8 +118,10 @@ def _read_fluid_selection(self, chunks, selector, fields, size):
                 for grid in chunk.objs:
                     nd = 0
                     for field in fields:
-                        ftype, fname = field
-                        position = f"/Timestep_{entry}/vars/{fname}"
+                        _, fname = field
+                        position = (
+                            f"/Timestep_{entry}/vars/{self.ds._field_name_map[fname]}"
+                        )
```

Comment:
Unless black is forcing this style (I don't think it does)
```suggestion
                        position = f"/Timestep_{entry}/vars/{self.ds._field_name_map[fname]}"
```

Reply:
alright, not critical